cloud-common/src/main/resources/application.yml

```diff
@@ -169,6 +169,7 @@ cb:
                 CDP 1.1 - Data Engineering: Apache Spark, Apache Hive, Apache Oozie=cdp-data-engineering-710;
                 CDP 1.1 - Data Engineering HA: Apache Spark, Apache Livy, Apache Zeppelin=cdp-data-engineering-ha-710;
                 CDP 1.1 - Data Mart: Apache Impala, Hue=cdp-data-mart-710;
+                CDP 1.1 - Active Data Mart: Apache Impala, Hue, Kudu, Spark=cdp-data-mart-active-710;
```

Comment:
Why "Apache" in "Apache Impala", but no "Apache" in Kudu and Spark?

Reply:
My bad; I forgot Hue wasn't an Apache project and assumed the "Apache" modifier applied to all of the listed projects. I'll fix this.